src/EditorFeatures/CSharpTest/QuickInfo/DiagnosticAnalyzerQuickInfoSourceTests.cs

```diff
@@ -161,28 +161,27 @@ public class C
             """, description, []);
     }
 
-    private static async Task AssertContentIsAsync(EditorTestWorkspace workspace, Document document, int position, string expectedDescription,
+    private static async Task AssertContentIsAsync(Document document, int position, string expectedDescription,
         ImmutableArray<TextSpan> relatedSpans)
     {
-        var info = await GetQuickinfo(workspace, document, position);
+        var info = await GetQuickInfo(document, position);
         var description = info?.Sections.FirstOrDefault(s => s.Kind == QuickInfoSectionKinds.Description);
         Assert.NotNull(description);
         Assert.Equal(expectedDescription, description.Text);
         Assert.Collection(relatedSpans,
             [.. info.RelatedSpans.Select(actualSpan => new Action<TextSpan>(expectedSpan => Assert.Equal(expectedSpan, actualSpan)))]);
     }
 
-    private static async Task<QuickInfoItem> GetQuickinfo(EditorTestWorkspace workspace, Document document, int position)
+    private static async Task<QuickInfoItem> GetQuickInfo(Document document, int position)
     {
-        var sharedGlobalCache = workspace.ExportProvider.GetExportedValue<DiagnosticAnalyzerInfoCache.SharedGlobalCache>();
```

Comment:
end goal (which will takea nother pr) is to stop exposing this type as something importable.